src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -24,6 +22,24 @@ const GovernanceSection = () => {
   const claimingApp = claimingSafeApp?.[0]
   const fetchingSafeClaimingApp = !claimingApp && !errorFetchingClaimingSafeApp
 
+  const WidgetLoadError = () => (
+    <Card className={css.loadErrorCard}>
+      <Box height="100%" display="flex" flexDirection="column" alignItems="center" justifyContent="center" gap={2}>
+        <LoadIcon fill={`${palette.primary.light}`} />
+        <Typography variant="body1" color="primary.light">
+          Couldn&apos;t load governance widget
+        </Typography>
+      </Box>
+    </Card>
+  )
+
+  const WidgetLoadErrorFallback = () => (
+    <Box className={css.loadErrorWrapper} display="flex" gap={3} width={1} sx={{ backgroundColor: 'background.main' }}>
+      <WidgetLoadError />
+      <WidgetLoadError />
```

Comment:
Why do we need two identical errors if it's the same app that failed to load?

Reply:
I think @liliiaorlenko idea here https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=1%3A2&t=I5ywjpeS3MXXV40t-0 is to display an error card per widget (despite being the same app)